src/main/java/eu/decentsoftware/holograms/api/utils/items/SkullUtils.java

```diff
@@ -118,17 +120,32 @@ public static String getTexture(@NonNull ItemStack itemStack) {
 	 * Set the Base64 texture of the given skull ItemStack.
 	 *
 	 * @param itemStack The ItemStack.
-	 * @param texture   The new skull texture (Base64).
+	 * @param texture   The new skull texture (Base64 or URL).
 	 */
 	public static void setSkullTexture(@NonNull ItemStack itemStack, @NonNull String texture) {
 		try {
 			ItemMeta meta = itemStack.getItemMeta();
 			if (meta instanceof SkullMeta) {
 				GameProfile profile = new GameProfile(UUID.randomUUID(), "");
-				Property property = new Property("textures", texture);
 
-				PropertyMap properties = profile.getProperties();
-				properties.put("textures", property);
+				String json;
+				
+				try{
+					json = Base64.getDecoder().decode(texture);
+				}
+				catch(IllegalArgumentException ex){
```

Comment:
Please, adhere to some unified code style. The formatting of this code is completely different from the rest of the project. This applies to all new code.